tensorflow/core/graph/mkl_tfconversion_pass.cc

```diff
@@ -77,6 +77,7 @@ class MklToTfConversionPass : public GraphOptimizationPass {
   // false, otherwise.
```

Comment:
Line 45, I think we can remove the usage of layer, which is kinds of confusing. We can consistently use node, and the an output node is usually referred as a target node or fetch node. The predecessor of TensorFlow, DistBelief, uses the concept of layer and TensorFlow avoided this concept on purpose and instead uses node and graph to represent more general-purpose computation structures.